### PR TITLE
Update README content guidelines [Fixes #515]

### LINKED ISSUE #515: Update Readme.md to reflect new site organization & sub-pages
**Is your feature request related to a problem? Please describe.**

The site recently changed its high-level organization into 3 categories: Individuals, Developers, and Enterprise. However, the ReadMe still reflects the old organization.

**Describe the solution you'd like**
Update the readme.md to reflect new organization, providing contributors with useful information about how to think about each sub-page.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by samajammin**
## Description
First swing at updating the README to reflect the new site content structure

## Related Issue
#515

### CODE REVIEW

**Comment by samajammin** [comment]
@jjmstark @CPSTL (& anyone else!) - please review when possible.

I'm definitely leaning towards moving to provide more general content guidelines vs. specific guidelines for each page, given the fact that more pages will be added in the future (e.g. #269) & that we'll continue to restructure site content (e.g. breaking out the /developers page into more targeted topics, similar to how we broke up /use into /eth, /wallets & /dapps in #643).

File: README.md [hunk 1]
```diff
@@ -81,36 +75,51 @@ Developer tool submissions will be assessed by the following criteria:
...
   - Has it been recently updated?
 - Is the tool widely used?
-  - We will consider metrics such as GitHub stars, download statistics, and whether it is used by known companies or projects.
+  - We will consider metrics such as GitHub stars, download statistics, and whether it is used by known companies or projects
 - Is the tool of sufficient quality?
   - Are there recurring bugs?
   - Is the tool reliable?
 
-### 📚 Learn
+When proposing new items to Ethereum.org we ask contributors to add their new addition(s) to the bottom of each section/list. This helps keep things fair for all contributors. In the future, we may consider adding a rankings system for the sections but for now this is the practice we are employing.
```
Comment (faraggi) [inline README.md]: ~~rankings system~~ > ranking system

**Comment by samajammin** [comment]
Notes from @jjmstark:

> My one comment is about the notes on specific pages

> I think you’re right this doesnt make sense any more, for the reasons you point out
and I think the solution is actually to get rid entirely of the “notes on specific pages” and replace it with “notes on specific kinds of contributions”

> so we should for instance roughly have a section which answers the following:
> - If I have a developer tool to add, where should i add it?
> - If I have a good article to add, where should I add it?
> - If I have a cool ethereum based application to add, where should I add it?

> some of that can re-use content that we already have, but it just shouldnt be framed as “Notes on the /developer page…”

**Comment by samajammin** [comment]
@jjmstark @CPSTL @faraggi - based on the feedback & reasons I mentioned above (i.e. the site continuing to expand & plans to break out pages into more targeted topics), I ended up removing page-specific guidelines & replacing them with topic-based guidelines:

- I want to add a developer tool or project
- I want to add an educational article or resource
- I want to add an Ethereum-based application

Please let me know what you think & if we should add any additional categories. Thanks!

**Pull request merged**